src/Stratis.Bitcoin.Features.Miner/Staking/PosMinting.cs

```diff
@@ -308,6 +308,14 @@ await this.GenerateBlocksAsync(walletSecret, this.stakeCancellationTokenSource.T
                     this.logger.LogDebug("Consensus error exception occurred in miner loop: {0}", cee.ToString());
                     this.rpcGetStakingInfoModel.Errors = cee.Message;
                 }
+                catch (ConsensusException ce)
+                {
+                    // All consensus exceptions should be ignored. It means that the miner
+                    // run into problems while constructing block or verifying it
+                    // but it should not halted the staking operation.
+                    this.logger.LogDebug("Consensus exception occurred in miner loop: {0}", ce.ToString());
```

Comment:
Probably log on error level

Reply:
Its similar ot the other exception trace levels